<ui version="4.0" >
 <class>QClutEditorDialogBase</class>
 <widget class="QDialog" name="QClutEditorDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CLUT editor</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QComboBox" name="m_clutPresetsComboBox" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_loadClutPushButton" >
       <property name="toolTip" >
        <string>Load transfer function</string>
       </property>
       <property name="text" >
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_saveClutPushButton" >
       <property name="toolTip" >
        <string>Save transfer function</string>
       </property>
       <property name="text" >
        <string>Save...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_switchEditorPushButton">
       <property name="text">
        <string>Switch editor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="m_editorsStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="udg::QTransferFunctionEditorByGradient" native="1" name="m_gradientEditor"/>
     <widget class="udg::QTransferFunctionEditorByValues" native="1" name="m_editorByValues"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::QTransferFunctionEditorByGradient</class>
   <extends>QWidget</extends>
   <header>qtransferfunctioneditorbygradient.h</header>
  </customwidget>
  <customwidget>
   <class>udg::QTransferFunctionEditorByValues</class>
   <extends>QWidget</extends>
   <header>qtransferfunctioneditorbyvalues.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QClutEditorDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QClutEditorDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
